index.js

```diff
@@ -31,14 +33,22 @@ Settings.loadRepositoriesSettings('./config.json')
             dateLog('jsonResponse: ' + jsonResponse);
 
             var promise = Promise.resolve();
+            var headers = {
+                'User-Agent': 'cesium-concierge',
+                Authorization: 'token ' + Settings.repositories[repositoryName].gitHubToken
+            };
+
             if ((event === 'issues' || event === 'pull_request') && jsonResponse.action === 'closed') {
                 promise = promise.then(function () {
-                    return commentOnClosedIssue(jsonResponse, {
-                        'User-Agent': 'cesium-concierge',
-                        Authorization: 'token ' + Settings.repositories[repositoryName].gitHubToken
-                    });
+                    return commentOnClosedIssue(jsonResponse, headers);
+                });
+            }
+            if (event === 'pull_request' && jsonResponse.action === 'opened') {
```

Comment:
That's what I meant.  I would change it so it's obvious to the reader that only one will be hit.